wdl/tasks/VariantCalling/GLNexus.wdl

```diff
@@ -227,13 +259,34 @@ task Call {
 
         echo ~{gvcfs[0]} | sed 's/.*locus_//' | sed 's/.g.vcf.bgz//' | sed 's/___/\t/g' > range.bed
 
+        gvcf_file_list=~{write_lines(gvcfs)}
+
+        if [[ "~{force_add_missing_dp}" == "true" ]]; then
```

Comment:
This should be moved to a separate task, given that this isn't the functionality of GLnexus, rather of bcftools.